jax/interpreters/mlir.py

```diff
@@ -1277,7 +1277,7 @@ def emit_python_callback(platform, callback,
       [aval_to_ir_types(aval) for aval in result_avals])
   result_type = ir.TupleType.get_tuple(result_types)
   call_target_name = ("xla_python_gpu_callback"
-                      if platform == "cuda" else "xla_python_cpu_callback")
+                      if platform == "cuda" or platform == "rocm" else "xla_python_cpu_callback")
```

Comment:
Could make this `if platform in {"cuda", "rocm"}`

Reply:
Done